### PR TITLE
Features => Production

### PR DESCRIPTION
**Pull request by carlos-ssh**
Hello, in this Pull Request we add these features to our project:
- Implement a basic UI with plain HTML.
- Create a collection that keeps a list of books.
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection.
- Display all books saved in the collection in the top part of the page.

When a user clicks the "Add" button:

1. - A new book is added to the collection.
2. - The new book is displayed in the page.

When a user clicks the "Remove" button:

1. The correct book is removed from the collection.
2. The correct book disappears from the page.

- Data is preserved in the browser's memory by using localStorage.

Cheers!!

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  library.push(book);
+  displayBooks();
+  saveLibrary();
+}
+
+function removeBook(title) {
+  library = library.filter((book) => book.title !== title);
+  displayBooks();
+  saveLibrary();
+}
```
Comment (sinansevgi) [inline app.js]: - [x]  The removeBook function doesn't work as correctly. You should also consider books with the same title (or I can have multiple copies of the same book). Let's say I have The Double from Fyodor Dostoevsky and The Double from José Saramago, this method will remove both of them when I try to remove one of them. 

The suggested way to solve this issue is to give unique ids for each book record and remove records with ids.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+function displayBooks() {
+  const libraryDiv = document.getElementById('library');
+  libraryDiv.innerHTML = '';
+  library.map((book) => {
+    const divBook = document.createElement('div');
+    const p = document.createElement('p');
+    p.innerHTML = book.author;
```
Comment (sinansevgi) [inline app.js:29]: [Optional]
If you're just changing the inner text, `textContent` would be more appropriate. ([And it's safer than `innerHTML`](https://blog.cloudboost.io/why-textcontent-is-better-than-innerhtml-and-innertext-9f8073eb9061)) 
You have lots of `innerHTML` in this function. (that only changes the inner text) You should consider changing them to `textContent`.
```suggestion
    p.textContent = book.author;
```

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @carlos-ssh , 

Good job so far!
`displayBooks` function works really great :clap: 
There are some issues that you still need to work on to go further but you are almost there!

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 2 above)
Comment (carlos-ssh) [reply app.js:29]: Ok, thanks.

**Review by carlos-ssh — COMMENTED** [review]
Added ID's to new books.

**Review by voscarmv — APPROVED** [review]
Hi @carlos-ssh !

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**